test/test_helper.rb

```diff
@@ -160,23 +161,18 @@ def assert_no_missing_translations(msg = "")
     ##
     # execute a block with a given set of HTTP responses stubbed
     def with_http_stubs(stubs_file)
-      http_client_save = OSM.http_client
-
-      begin
-        stubs = YAML.load_file(File.expand_path("../http/#{stubs_file}.yml", __FILE__))
+      stubs = YAML.load_file(File.expand_path("../http/#{stubs_file}.yml", __FILE__))
+      stubs.each do |url, response|
+        stub_request(:get, Regexp.new(Regexp.quote(url))).to_return(:status => response["code"], :body => response["body"])
```

Comment:
If we're turning the URL into a regex do we need to add anchors to the start and end so we don't accidentally get a substring match? I realise it's pretty unlikely but anchoring it makes it more correct I think?